io.catenax.user_estimated_loading/1.0.0/UserEstimatedLoading.ttl

```diff
@@ -0,0 +1,248 @@
+   bamm:preferredName "History profiles entity"@en ;
+   bamm:description "Collection of previous usage profiles "@en ;
+   bamm:properties ( :percentage [ bamm:property :trailerUse; bamm:optional true ] [ bamm:property :cityDriving; bamm:optional true ] :drivingStyle :hilliness :curviness ) .
+
+:FutureScenarioEntity a bamm:Entity ;
+   bamm:preferredName "Future scenario list entity"@en ;
+   bamm:description "List of future scenarios"@en ;
+   bamm:properties ( :futureScenario :futureScenarioId ) .
+
+:mileage a bamm:Property ;
+   bamm:preferredName "Mileage @en"@en ;
+   bamm:description "Distance the vehicle has travelled in kilometre"@en ;
+   bamm:characteristic :Mileage ;
+   bamm:exampleValue "123456"^^xsd:positiveInteger .
+
+:ageInMonth a bamm:Property ;
+   bamm:preferredName "Vehicle age"@en ;
+   bamm:description "Age of the vehicle in month"@en ;
+   bamm:characteristic :AgeInMonths ;
+   bamm:exampleValue "42"^^xsd:nonNegativeInteger .
+
+:vehicleType a bamm:Property ;
+   bamm:preferredName "Vehicle variant"@en ;
+   bamm:description "Vehicle variant - body shapes like SUV, Van, Limousine"@en ;
+   bamm:characteristic :KbaVariant ;
+   bamm:exampleValue "SUV" .
+
+:numberOfDriveAxle a bamm:Property ;
+   bamm:preferredName "Number of drive axles"@en ;
+   bamm:description "Identification of 4WD, default 1: 2WD"@en ;
+   bamm:characteristic :NumberOfDriveAxle ;
+   bamm:exampleValue "2"^^xsd:positiveInteger .
+
+:energySourceCode a bamm:Property ;
+   bamm:preferredName "Energy source code"@en ;
+   bamm:description "The code for the energy source respectively fuel type of the vehicle according to KBA list"@en ;
+   bamm:characteristic :EnergySource ;
+   bamm:exampleValue "0002" .
+
+:powerKW a bamm:Property ;
+   bamm:preferredName "Power @en"@en ;
+   bamm:description "Engine power is the power that an engine can put out in kW"@en ;
+   bamm:characteristic :EnginePowerKW ;
+   bamm:exampleValue "145"^^xsd:positiveInteger .
+
+:grossVehicleWeightKG a bamm:Property ;
+   bamm:preferredName "Gross vehicle weight"@en ;
+   bamm:description "Technical permissable maximum weight of the vehicle in kg"@en ;
+   bamm:characteristic :Weight .
+
+:percentage a bamm:Property ;
+   bamm:preferredName "Weightfactor in percantage"@en ;
+   bamm:description "Weighfactor of the different profiles, a user has. Different profiles indicates different user as well as different usages of like profiles for daily work routes or for caravan holiday."@en ;
+   bamm:characteristic :Percentage ;
+   bamm:exampleValue "23.5"^^xsd:decimal .
+
+:trailerUse a bamm:Property ;
+   bamm:preferredName "Trailer use switch"@en ;
+   bamm:description "Usage of trailer like caravan. "@en ;
+   bamm:characteristic :Boolean ;
+   bamm:exampleValue "true"^^xsd:boolean .
+
+:cityDriving a bamm:Property ;
+   bamm:preferredName "City driving switch"@en ;
+   bamm:description "City driving means a short distances."@en ;
+   bamm:characteristic :Boolean ;

```

Comment:
for some strange reason the meta model version of the boolean characteristic was replaced by a custom one in your code. It should be 
   bamm:characteristic bamm-c:Boolean;
otherwise several validation problems occur